<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetWorkConfigDialog</class>
 <widget class="QDialog" name="NetWorkConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labHostAddress">
     <property name="text">
      <string>主机地址:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbHostAddress"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>端口:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditPort"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnClearHost">
     <property name="text">
      <string>清除记录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnHostOK">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnHostCancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
